1.fee_bumping/fee_bumping.md

```diff
@@ -153,33 +153,224 @@ Bitcoin blockchain, such as exchanges or custodians:
+transaction in a node's mempool will be replaced. BIP 125 is referred to as
+*opt-in* RBF since the node will only allow a transaction to be replaced if
+the sender has explicitly signaled that he wants the transaction to be
+replacable (or if the transaction is the descendant of an unconfirmed
+transaction signaling replaceability).
+
+> WARNING: transaction replacement policies are outside the domain of consensus.
+> A transaction not being marked as replaceable offers no guarantees that it won't
+> be double-spent. Other nodes on the network may have different replacement
+> policies and miners are free to select whatever transactions they want in
+> blocks.
+
+The spender signals that a transaction is replaceable by setting the `nSequence`
+number on any of its inputs to less than (0xffffffff - 1). A transaction is
+replaceable if it fulfils [five conditions][]. The BIP documents those condtions,
+but the implications of those conditions may not be immediately obvious. The next
+section describes some of those implications.
+
+[five conditions]: https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki#implementation-details
+
+### Implications of the conditions for BIP 125 RBF
+
+> One or more transactions currently in the mempool (original transactions)
+> will be replaced by a new transaction (replacement transaction) that spends
+> one or more of the same inputs if,
+
+- implication 1: RBF can be used to replace multiple transactions at the same
+  time. That could either be because the replacement transaction is directly
+  replacing two or more original tranasactions by spending their inputs, or
+  because the original transaction that is being replaced has descendants in
+  the mempool which will also need to be removed.
+- implication 2: Additional inputs can be added to the replacement transaction.
+  This is useful if a wallet needs to bump the fee on a transaction but there
+  is no change output to subtract the additional fee from.
+- implication 3: The replacement transaction does not need to include all of
+  the inputs from the original transaction. This can be extremely dangerous
+  since wallets can easily overspend this way. For example, if the original
+  transaction uses inputs A and B, a replacement transaction is sent with
+  inputs B and C, and then that replacement transaction is replaced with a
+  new replacement transaction using inputs C and D, the first transaction
+  could be rebroadcast by a third party at any point in the future, and both
+  would be valid! Wallets are advised to include all inputs from the original
+  transaction in the replacement transaction unless they have very reliable
+  tracking to make sure that they don't double-spend themselves.
+- implication 4: the `nSequence` of the inputs in the replacement transaction
+  are not considered, so a replaceable transaction could be replaced by a
+  non-replaceable transaction. If a wallet wants to be able to replace the
```

Comment:
How is the replacing with a non-replaceable transaction compatible with the inheritance of replaceability mentioned above?